### ISSUE TITLE
Possible integration with NextScripts Social Network Auto Poster plugin

### ISSUE THREAD

**Issue by RyanBayne** [RyanBayne/TwitchPress#105]
This is a possible integration to aid sharing of a post to other social networks. We will either integrate the core plugin to work with NextScripts Social Network Auto Poster plugin or provide similar functionality. Research has to be done first. 

Such features need to work with other post types and not only the TwitchPress one.

**Comment by ThatGirlSlays** [comment]
Just to be clear on this, I am not looking for any integration with posting Twitch Press content to social media such as Twitter/Facebook. What I'm referring to is how NextScripts' plugin, when posting certain post types, provides a check box on the Add/Edit post page. If the box is checked, then their plugin posts to whatever social media accounts you previously set up.

What I'd be interested in for Twitch Press, is when someone creates a post (or other custom post type), they have the option to check a box that will then share that post on their Twitch channel feed.

**Comment by RyanBayne** [comment]
The original issue will stay as is and I will take a moment to consider possible integrations.

But your comment clears up my confusion. The feature you ask for would be displayed on posts or custom posts but not the TwitchPress custom post type. 

I'll make another issue for it and copy paste your comment. 

Thanks again for your effort.

**Comment by RyanBayne** [comment]
This task is related to a feature Twitch.tv no longer provides.
